docs/products/kafka/howto/integrate-service-logs-into-kafka-topic.rst

```diff
@@ -0,0 +1,36 @@
+Integration of logs into Apache Kafka topic
+============================================
+
+You can send logs from your services into a specified Apache Kafka topic. The setup can be done through `Aiven console <https://console.aiven.io>`_.
+
+.. note::
+
+    Note, the integration can be used for both Aiven for Apache Kafka, as well as external Kafka clusters registered in the project's service integrations page.
+
+In this example we'll show how to send the logs from an Aiven for PostgreSQL service to a topic in your Aiven for Apache
+Kafka service. For this example you need:
+
+-  a created and running Aiven for PostgreSQL service (we'll refer to it as *a source*)
+-  a created and running Aiven for Apache Kafka service (we'll refer as *a destination*)
+
+Prepare the destination to receive the logs
+-------------------------------------------
+
+In the Aiven for Apache Kafka overview page enable Kafka REST API and create a Kafka topic where you want to receive the logs.
+
+Add a new integration to the source
+-----------------------------------
+
+Go to the overview page of your PostgeSQL service, scroll the list of actions till you see **Service integrations** and click on **Manage Integrations**. A new pop up window will be displayed showing a list of available integrations for your service. Select **Kafka Logs** from this list.
```

Comment:
Typo PostgeSQL (it can't pick up things it thinks are an acronym - but we could add it to the suggestions list)